app-admin/ansible-lint/ansible-lint-4.3.6.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{6..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Checks ansible playbooks for practices and behaviour that can be improved"
+HOMEPAGE="https://github.com/ansible/ansible-lint"
+SRC_URI="https://github.com/ansible/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+CDEPEND="app-admin/ansible[${PYTHON_USEDEP}]
+		dev-python/pyyaml[${PYTHON_USEDEP}]
+		dev-python/six[${PYTHON_USEDEP}]"
+DEPEND="dev-python/rich[${PYTHON_USEDEP}]
+		dev-python/ruamel-yaml[${PYTHON_USEDEP}]
+		dev-python/setuptools[${PYTHON_USEDEP}]
+		dev-python/setuptools_scm[${PYTHON_USEDEP}]
+		dev-python/setuptools_scm_git_archive[${PYTHON_USEDEP}]
+		test? (
+			dev-python/flake8[${PYTHON_USEDEP}]
+			dev-python/nose[${PYTHON_USEDEP}]
+			dev-python/wheel[${PYTHON_USEDEP}]
+			"$(python_gen_cond_dep 'dev-python/typing-extensions[${PYTHON_USEDEP}]' python3_{6,7} )"
```

Comment:
I based this off of my understanding of https://github.com/ansible/ansible-lint/blob/a4e6409cb2de8015299ad58fdb83b1bf3faf9a72/setup.cfg#L79